### PR TITLE
mysql_user: fix documentation

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
mysql_user: fix documentation

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
`plugins/modules/mysql_user.py`

### CODE REVIEW

File: plugins/modules/mysql_user.py [hunk 1]
```diff
@@ -118,12 +119,13 @@
     version_added: '0.1.0'
 
 notes:
-   - "MySQL server installs with default login_user of 'root' and no password. To secure this user
-     as part of an idempotent playbook, you must create at least two tasks: the first must change the root user's password,
-     without providing any login_user/login_password details. The second must drop a ~/.my.cnf file containing
-     the new root credentials. Subsequent runs of the playbook will then succeed by reading the new credentials from
-     the file."
+   - "MySQL server installs with default login_user of 'root' and no password.
```
Comment (felixfontein) [inline plugins/modules/mysql_user.py]: How about using `I(...)` for `login_user` and `login_password`? And `C(...)` for `'root'`?

Comment (Andersson007) [reply plugins/modules/mysql_user.py]: Sounds good! Fixed, thanks!

File: plugins/modules/mysql_user.py [hunk 2]
```diff
@@ -118,12 +119,13 @@
...
-     as part of an idempotent playbook, you must create at least two tasks: the first must change the root user's password,
-     without providing any login_user/login_password details. The second must drop a ~/.my.cnf file containing
-     the new root credentials. Subsequent runs of the playbook will then succeed by reading the new credentials from
-     the file."
+   - "MySQL server installs with default login_user of C(root) and no password.
+     To secure this user as part of an idempotent playbook, you must create at least two tasks:
+     1) change the root user's password, without providing any I(login_user)/I(login_password) details,
+     2) drop a C(~/.my.cnf) file containing the new root credentials.
+     Subsequent runs of the playbook will then succeed by reading the new credentials from the file."
    - Currently, there is only support for the `mysql_native_password` encrypted password hash module.
```
Comment (felixfontein) [inline plugins/modules/mysql_user.py]:
```suggestion
   - Currently, there is only support for the C(mysql_native_password) encrypted password hash module.
```

File: plugins/modules/mysql_user.py [hunk 3]
```diff
@@ -118,12 +119,13 @@
     version_added: '0.1.0'
 
 notes:
-   - "MySQL server installs with default login_user of 'root' and no password. To secure this user
-     as part of an idempotent playbook, you must create at least two tasks: the first must change the root user's password,
-     without providing any login_user/login_password details. The second must drop a ~/.my.cnf file containing
-     the new root credentials. Subsequent runs of the playbook will then succeed by reading the new credentials from
-     the file."
+   - "MySQL server installs with default login_user of C(root) and no password.
```
Comment (felixfontein) [inline plugins/modules/mysql_user.py]:
```suggestion
   - "MySQL server installs with default I(login_user) of C(root) and no password.
```

**Review by felixfontein — APPROVED** [review]

File: plugins/modules/mysql_user.py (hunk 3 above)
Comment (Andersson007) [reply plugins/modules/mysql_user.py]: @felixfontein if we go further and further to this .."old" module, i'm afraid, we'll never finish:)
committed both, thanks!

**Comment by Andersson007** [comment]
@felixfontein thanks for reviewing:)

**Pull request merged**